recipes/hdmf_zarr/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "hdmf_zarr" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 281cfcd1581ed8e101953015179aedc01e55740980540f7b0e43f67110fa44f5
+  
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - hdmf ==3.4.0
+    - zarr ==2.11.0
+    - numcodecs ==0.9.1
+    - pynwb ==2.0.0
+    - setuptools
+
+test:
```

I would recommend adding the `pip check` test back in.

---

Can you add as was requested here?
```suggestion
test:
  commands:
    - pip check
```